App.tsx

```diff
@@ -32,13 +51,19 @@ export default function App() {
     <NavigationContainer>
       <Stack.Navigator>
         <Stack.Screen
-          name="Home"
+          name="BottomTabs"
```

Comment:
You already have a `Home` screen in the `TabNavigation`. All screens should have unique names.